app/migrations/20180518_uphold.js

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const compareVersions = require('compare-versions')
+
+// per https://github.com/brave/browser-laptop/issues/14152
+// add fingerprint exception for existing users for uphold.com
+module.exports = (data) => {
+  // don't apply if:
+  // - user chooses to block all fingerprinting (global setting)
+  // - user is not upgrading from 0.22.714 or earlier
+  if (data.fingerprintingProtectionAll || !data.lastAppVersion) {
```

Comment:
i think you want to check `data.fingerprintingProtectionAll.enabled`?

Reply:
After missing this, I fixed this and added some unit tests! 😄 Should cover most of the cases